### DISCUSSION TITLE
How can I display all the error message from the register using useEffect in react hook form

### DISCUSSION THREAD

**Discussion by nani1008** [react-hook-form/react-hook-form#4210] [Q&A]
What I want to do is display all the error message as long as the page loads using useEffect. Then I have a 3 error messages that will be displayed if the input is focused beside these text theres an icon “check” and “x” if the validation is met it is “check” then if its not its “x”. Heres a reference for what I want to achieve. Thanks for your help. https://www.w3schools.com/howto/tryit.asp?filename=tryhow_js_password_val

**Comment by bluebill1049** [comment]
`criteriaMode: "all"`

**Reply by nani1008** [reply]
im almost done with it but my problem now is on the first load of the page the error is "validation met" what I want is on the first load of the page the error is "error message" https://codesandbox.io/s/angry-moon-higue?file=/src/App.js here is an example I made for what I want to do. thank youu!!!!

**Reply by bluebill1049** [reply]
you can read `isValid` formstate right?
